server/jetstream_errors.go

```diff
@@ -0,0 +1,220 @@
+package server
+
+import (
+	"fmt"
+)
+
+type APIErrorIdentifier int
+
+const (
+	JSUnknownErr APIErrorIdentifier = iota
+	JSAccountResourcesExceededErr
```

Comment:
I kind of want to generate this file rather than hand maintain it. Given a source - json or yaml - that roughly looks like the structure a bit lower `go generate` would make this file.

The benefit of this is that I could automatically add a comment here above each of these identifiers that include the Description from the `ApiErrors` struct, this will greatly help code use because when you hover your mouse over the error you'll get some feedback about its body and such, now using these are a bit of pain.  

I doubt yous will like `go generate` based flow though.

Reply:
I am not a fan of it as a main build flow, but we could use it and check in the resulting files.

Reply:
yes, that would be the idea - use it only when the source file changes and check in the result

Reply:
Discussed with Ivan, we'll go with generate